src/Components/Display-feed/DisplayFeed.css

```diff
@@ -22,22 +30,40 @@
     }
 }
 
+@media screen and (min-width: 768px) {
+    .display-feed {
+        margin-bottom: 3em;
+    }
+}
+
 @media screen and (min-width: 80em) {
     .display-feed {	
         padding-bottom: 5em;	
     }
     .img-container {
         width: 80%;
-        max-width: 1010px;
         margin: 0 auto;
         grid-template-columns: repeat(6, 1fr);
     }
+    .img-container li {
+        grid-column: span 2;
+        grid-row: span 2;
+    }
+}
+
+@media screen and (min-width: 1440px) {
+    .img-container {
+        max-width: 990px;
+        margin: calc((100vw - 990px)/2);
+    }
```

Comment:
These changes cause a huge bit of padding: https://share.getcloudapp.com/BluNBRNy

Reply:
Good catch. What was the intent of the calc, do we know?